src/visuals.js

```diff
@@ -25,17 +24,17 @@ export const Visuals = {
     if (!this.showIt) return
 
     // when there are 3 or more bodies, step the simulation
-    if (this.bodies.filter(b => !b.life || b.life > 0).length >= 3) {
+    if (this.bodies.filter((b) => !b.life || b.life > 0).length >= 3) {
```

Comment:
not sure why prettier has fixes in here again

Reply:
i also tried turning off all of my format-on-save settings and just let prettier take over but haven't noticed consistency

Reply:
I recommend installing the prettier plugin to format on save.

Reply:
I think this was from merging in some pre-prettier stuff. 🤔🧐